server/internal/controllers/chain.go

```diff
@@ -379,34 +380,13 @@ LIMIT 1
 			c.String(http.StatusInternalServerError, "User could not be added to chain due to unknown error")
 			return
 		}
-
-		// find admin users related to the chain to email
-		results, err := models.UserGetAdminsByChain(db, chain.ID)
+		err := services.EmailLoopAdminsOnUserJoin(db, user, chain.ID)
 		if err != nil {
-			goscope.Log.Errorf("Error retrieving chain admins: %s", err)
-			c.String(http.StatusInternalServerError, "No admins exist for this loop")
-			return
-		}
-		if len(results) == 0 {
-			goscope.Log.Errorf("Empty chain that is still public: ChainID: %d", chain.ID)
-			c.String(http.StatusInternalServerError, "No admins exist for this loop")
+			goscope.Log.Errorf("Unable to send email to associated loop admins: %v", err)
```

Comment:
if the 500 is raised here, the user never receives a message, but the DB is already updated. And if they return, the signup is "waiting for approval". Maybe it's better to at least send the email to the user in all cases?